src/ConfigurationTransform/Services/Extensions/ProjectItemExtensions.cs

```diff
@@ -38,19 +38,31 @@ public static string GetFullPath(this ProjectItem source)
 
         public static ProjectItem ParentProjectItemOrDefault(this ProjectItem source)
         {
-            var containingProject = source.ContainingProject;
-            if (containingProject == null) return null;
+            // getting the parent of the clicked project item
+            var parent = (ProjectItem) source.Collection.Parent;
```

Comment:
What if the `source.Collection.Parent`is `Project `and not `ProjectItem`?
You are going to get cast exception.